src/audio/google_rtc_audio_processing.c

```diff
@@ -325,7 +460,7 @@ static int google_rtc_audio_processing_prepare(struct comp_dev *dev)
 
 		if (source_c->source->pipeline->pipeline_id != dev->pipeline->pipeline_id) {
 			cd->aec_reference = source;
-			aec_channels = sourcs_c->stream.channels;
```

Comment:
this should be its own commit so it can be cherrypicked without your change if needed

Reply:
Done. Reverted my change.